spark/v3.2/build.gradle

```diff
@@ -17,28 +17,28 @@
  * under the License.
  */
 
-def sparkProjects = [
-    project(':iceberg-spark:iceberg-spark-3.2_2.12'),
-    project(":iceberg-spark:iceberg-spark-extensions-3.2_2.12"),
-    project(':iceberg-spark:iceberg-spark-runtime-3.2_2.12')
+String sparkVersion = '3.2.0'
+String sparkMajorVersion = '3.2'
+String scalaVersion = System.getProperty("scalaVersion") != null ? System.getProperty("scalaVersion") : System.getProperty("defaultScalaVersion")
+
+def projects = [
+        project(":iceberg-spark:iceberg-spark-${sparkMajorVersion}_${scalaVersion}"),
+        project(":iceberg-spark:iceberg-spark-extensions-${sparkMajorVersion}_${scalaVersion}"),
+        project(":iceberg-spark:iceberg-spark-runtime-${sparkMajorVersion}_${scalaVersion}"),
 ]
 
-configure(sparkProjects) {
-  project.ext {
-    sparkVersion = '3.2.0'
-  }
-
+configure(projects) {
```

Comment:
Can you revert the change that renames sparkProjects to projects? That way we have fewer conflict opportunities.